src/homepageExperience/components/Navigation.tsx

```diff
@@ -1,16 +1,58 @@
 import React, {PureComponent} from 'react'
-import classnames from 'classnames'
-
-import {InfluxColors} from '@influxdata/clockface'
-
-import {OverviewIcon} from 'src/homepageExperience/components/HomepageIcons'
+import {
+  AlertIcon,
+  ExecuteQueryIcon,
+  FinishIcon,
+  InitializeClientIcon,
+  InstallDependenciesIcon,
+  OverviewIcon,
+  TokenIcon,
+  WriteDataIcon,
+} from 'src/homepageExperience/components/HomepageIcons'
+import Step from './steps/Step'
 
 interface OwnProps {
   currentStep: number
 }
 
-const defaultFillColor = InfluxColors.Grey95
-const activeFillColor = '#0098f0'
+const steps = [
+  {
+    name: 'Overview',
+    icon: OverviewIcon,
+  },
+  {
+    name: 'Install \n Dependencies',
+    icon: InstallDependenciesIcon,
+  },
+  {
+    name: 'Create a \n Token',
+    icon: TokenIcon,
+  },
+  {
+    name: 'Initialize \n Client',
+    icon: InitializeClientIcon,
+  },
+  {
+    name: 'Write \n Data',
+    icon: WriteDataIcon,
+  },
+  {
+    name: 'Execute a \n Simple Query',
+    icon: ExecuteQueryIcon,
+  },
+  {
+    name: 'Execute an \n Aggregate Query',
+    icon: ExecuteQueryIcon,
+  },
+  {
+    name: '(Optional) \n Set up Alerts',
+    icon: AlertIcon,
+  },
+  {
+    name: 'Finish',
+    icon: FinishIcon,
+  },
+]
```

Comment:
In case we want to add or remove more steps in future, this will make it easier.